src/screens/DiscoveryScreenV2/elements/DiscoveryItemList.js

```diff
@@ -25,19 +25,26 @@ const DomainList = (props) => {
     isHashtag,
     handleSetFollow,
     handleSetUnFollow,
+    handleSetBlock,
+    handleSetUnblock,
     onPressBody,
     DefaultImage,
-    isCommunity
+    isCommunity,
+    isBlockedSection
   } = props;
 
   return (
     <View style={styles.container}>
-      <TouchableOpacity onPress={() => onPressBody(item)} style={styles.wrapProfile}>
+      <TouchableOpacity
+        testID="pressbody"
+        onPress={() => onPressBody(item)}
+        style={styles.wrapProfile}>
         <View style={styles.card}>
           {!isHashtag ? (
             <React.Fragment>
               {item.image && typeof item.image === 'string' && item.image.length > 0 ? (
                 <Image
+                  testID="images"
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `Image` component is missing a `testID` prop for testing. It's good practice to include testIDs for all interactive components.
```diff
                 <Image
+                   testID="image"
                  source={{
                     uri: item.image
                   }}
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->